packages/core/src/indexing/sqlite/sqlite-index-api.ts

```diff
@@ -26,22 +26,39 @@ export class SqliteIndexApi implements DatabaseIndexApi {
     this.insertionOrder = new InsertionOrder(dbConnection)
   }
 
+  public getActiveModelsToIndex(): Array<StreamID> {
+    // Helper function to get array of active models that are currently being indexed by node
+    // TODO: update to runtime check once adminAPI unlocks to load models on the fly
+    return this.modelsToIndex
+  }
+
+  private _isIndexStream(args: StreamID): Boolean {
```

Comment:
`_isIndexedStream`?  `_shouldIndexStream`?

Reply:
I like it. will update.